test/Test_Elaboration.re

```diff
@@ -7,74 +7,139 @@ let dhexp_typ = testable(Fmt.using(Exp.show, Fmt.string), DHExp.fast_equal);
 
 let ids = List.init(12, _ => Id.mk());
 let id_at = x => x |> List.nth(ids);
+
+//Starts at 0
+let get_id_menhir_closure = (id_index: int) => {
```

Comment:
This seems unused

Reply:
Yes, sorry. That is unused - that's a relic from the DHExp (instead of universal Exp) where I had to generate ids. I'll delete it